### ISSUE TITLE
font sizes are very small

### ISSUE THREAD

**Issue by fn3k4** [docker/kitematic#5303]
I cannot see the text in the app. the font size is very very small and it cannot be scaled by settings. It makes the app useless for me.

**Comment by Trackhe** [comment]
@fn3k4 have you find a alternative or a fix for you?

**Comment by fn3k4** [comment]
@Trackhe , yes. I have unpacked kitematic\resources\app.asar (by command "asar d app.asar app") and added attribute "zoom" to "body" tag in app\main.css. Like that:
`
body {
  zoom: 150%;
`
asar packer/unpacker installes by node package manager - npm.

**Comment by Trackhe** [comment]
When you want I can add the option for you. In my build.
